### ISSUE TITLE
Build error in v1.41.0 with htslib in Linux

### ISSUE THREAD

**Issue by emsisson** [vgteam/vg#3688]
Greetings,

I am trying to build vg v1.41.0 in RHEL 7.9 using cmake 3.22.2 and gcc 9.3.0 but have hit an error.  I ran

$ . ./source_me.sh
$ nohup make 1> make.log 2>&1 &

and it exited with the following messages at the end of the log file while compiling deps/htslib/htscodecs/htscodecs/htscodecs.c:

htscodecs/htscodecs/htscodecs.c: In function ‘htscodecs_version’:
htscodecs/htscodecs/htscodecs.c:43:12: error: ‘HTSCODECS_VERSION_TEXT’ undeclared (first use in this function); did you mean ‘HTSCODECS_VERSION’?
   43 |     return HTSCODECS_VERSION_TEXT;
      |            ^~~~~~~~~~~~~~~~~~~~~~
      |            HTSCODECS_VERSION
htscodecs/htscodecs/htscodecs.c:43:12: note: each undeclared identifier is reported only once for each function it appears in
htscodecs/htscodecs/htscodecs.c:44:1: warning: control reaches end of non-void function [-Wreturn-type]
   44 | }
      | ^
make[1]: *** [htscodecs/htscodecs/htscodecs.o] Error 1
make[1]: Leaving directory `/var/tmp/vg/vg-v1.41.0/deps/htslib'
make: *** [lib/libhts.a] Error 2

This appears to be htslib-1.12. I have downloaded that separately and built it without problem. Comparing the two build environments, I see that header file version.h was not built in vg-v1.41.0 alongside htscodecs.c. How should I fix this?

Regards,
Eric Sisson

**Comment by faithokamoto** [comment]
We've gone through a bunch of releases, build changes, etc. since. In particular now our GCC minimum version is 10. If this or a similar problem occurs for a recent vg version please open a new Issue.
